src/app/view/variant/variant-view/variant-history/version-entry-layout/version-entry-layout.component.scss

```diff
@@ -70,3 +70,22 @@
     border: none;
   }
 }
+
+
+.user-id {
+  width: 140px;             /* Set the desired width */
+  overflow: auto;         /* Hide overflowing content */
+  text-overflow: ellipsis;  /* Display ellipsis (...) for overflow */
```

Comment:
As the input field doesn't show "asupersuperlongem..."but "asupersuperlongem", you might just comment that the email overflows, but doesn't display "..."

![image](https://github.com/user-attachments/assets/e0f88334-8152-4dbc-bc45-11784af71e9a)

Reply:
I have overworked the "UX concept" - now the email address gets copied and the log service shows a toast to inform you that the email address was copied. The copied function is as well active, when user clicks on the comments.